flink-yarn/src/test/java/org/apache/flink/yarn/YarnFileStageTestS3ITCase.java

```diff
@@ -161,10 +161,8 @@ private void testRecursiveUploadForYarn(String scheme, String pathSuffix) throws
 		assumeFalse(fs.exists(basePath));
 
 		try {
-			final Path directory = new Path(basePath, pathSuffix);
-
 			YarnFileStageTest.testCopyFromLocalRecursive(fs.getHadoopFileSystem(),
-				new org.apache.hadoop.fs.Path(directory.toUri()), Path.CUR_DIR, tempFolder, false);
+				Path.CUR_DIR, tempFolder, false);
```

Comment:
@jiasheng55 The `YarnFileStageTestS3ITCase .testRecursiveUploadForYarnS3a ()` seems to be failing on my Travis and the failure seems to be related to this change here. Could you please fix it and then I can merge.

Reply:
@kl0u Sorry I didn't reproduce the failure on my Travis and I don't know how to run `YarnFileStageTestS3ITCase` manually, could you tell me how to reproduce this failure?

Reply:
@jiasheng55 in general we are trying to avoid using `mockito` in our tests, so I suggest to try to get rid of it here.